scripts/animatediff_infv2v.py

```diff
@@ -157,9 +157,10 @@ def mm_sd_forward(self, x_in, sigma_in, cond):
                     mm_cn_restore(_context)
                 return x_out
 
-            logger.info("inner model forward hooked")
-            cfg_params.denoiser.inner_model.original_forward = cfg_params.denoiser.inner_model.forward
-            cfg_params.denoiser.inner_model.forward = MethodType(mm_sd_forward, cfg_params.denoiser.inner_model)
+            if getattr(cfg_params.denoiser.inner_model, 'original_forward', None) is None:
```

Comment:
why do we need this change?

Reply:
As the description before: the FreeInit iteration repeatedly calls `cfg_denoiser.forward` and so as the callback function `animatediff_on_cfg_denoiser()`. To prevent the `original_forward` from being discarded in the second call of `animatediff_on_cfg_denoiser()`, I introduced a getattr() check.